topics/imaging/tutorials/voronoi-segmentation/tutorial.md

```diff
@@ -0,0 +1,312 @@
+This can be achieved with the following operation. 
+> <hands-on-title> Task description </hands-on-title>
+> Combine the tesselation with the seeds and the mask to generate a segmentation that limits the expanse of each segment: 
+> 1. {% tool [Process images using arithmetic expressions with numpy](toolshed.g2.bx.psu.edu/repos/imgteam/image_math/image_math/1.26.4+galaxy2) %} with the following parameters:
+>    - *"Expression"*: `tessellation * (mask / 255) * (1 - seeds / 255)`
+>    - In *"Input images"*:
+>        - {% icon param-repeat %} *"Insert Input images"*
+>            - {% icon param-file %} *"Input Image"*: `tesselation`
+>            - *"Variable for representation of the image within the expression"*: `tesselation`
+>        - {% icon param-repeat %} *"Insert Input images"*
+>            - {% icon param-file %} *"Input Image"*: `seeds`
+>            - *"Variable for representation of the image within the expression"*: `seeds`
+>        - {% icon param-repeat %} *"Insert Input images"*
+>            - {% icon param-file %} *"Input Image"*: `mask`
+>            - *"Variable for representation of the image within the expression"*: `mask`
+>
+> 2. Rename the output to `masked segmentation`.
+>
+> 3. {% tool [Colorize label map](toolshed.g2.bx.psu.edu/repos/imgteam/colorize_labels/colorize_labels/3.2.1+galaxy3) %}. 
+>    - {% icon param-file %} *"Input Image"*: `masked segmentation`
+>    - *"Radius of the neighborhood"*: `10` (works well for the HeLa cell image; may need adjustment for the tree image)
+>    - *"Background label"*: `0`
+> 
+> 4. Rename the output to `colorized label map`
+>
+> 5. {% tool [Convert single-channel to multi-channel image](toolshed.g2.bx.psu.edu/repos/imgteam/repeat_channels/repeat_channels/1.26.4+galaxy0) %} with: 
+>    - {% icon param-file %} *"Input Image"*: `single channel image`
+>    - *"Number of channels"*: `3`
+> 
+> 6. Rename the output to `multi channel image`
+> <!-- channel wrangling end -->
+> 
+> 7. {% tool [Overlay images](toolshed.g2.bx.psu.edu/repos/imgteam/overlay_images/ip_overlay_images/0.0.4+galaxy4) %} with the following parameters to overlay the Voronoi segmentation on the original image:  
+>    - *"Type of the overlay"*: `Linear blending`
+>    - {% icon param-file %} *"Image 1"*: `multi channel image`
+>    - {% icon param-file %} *"Image 2"*: `colorized label map`
+>    - *"Weight for blending"*: `0.5`
+>
+{: .hands_on}
+
+
+## Count objects and extract image features
+> <hands-on-title> Task description </hands-on-title>
+> 1. {% tool [Count objects in label map](toolshed.g2.bx.psu.edu/repos/imgteam/count_objects/ip_count_objects/0.0.5-2) %}. 
+>    - {% icon param-file %} *"Input Image"*: `tesselation`
+>
+> 1. {% tool [Extract image features with scikit-image](toolshed.g2.bx.psu.edu/repos/imgteam/2d_feature_extraction/ip_2d_feature_extraction/0.18.1+galaxy0) %} with the following parameters:
```

Comment:
The name of the tool is just `Extract image features`, the `with scikit-image` part is the description:
```suggestion
> 1. {% tool [Extract image features](toolshed.g2.bx.psu.edu/repos/imgteam/2d_feature_extraction/ip_2d_feature_extraction/0.18.1+galaxy0) %} with the following parameters:
```